src/main/java/com/conveyal/datatools/manager/jobs/ProcessSingleFeedJob.java

```diff
@@ -130,7 +130,8 @@ public void jobLogic() {
             snapshot.feedTransformResult = dbTarget.feedTransformResult;
             // If the user has selected to create a new version from the resulting snapshot, do so here.
             if (rules.createNewVersion) {
-                addNextJob(new CreateFeedVersionFromSnapshotJob(feedSource, snapshot, owner));
+                // Publishing the proprietary files will preserve the pattern names in the newly created feed version.
+                addNextJob(new CreateFeedVersionFromSnapshotJob(feedSource, snapshot, owner, true));
```

Comment:
This feels like a scary new default. How are we sure this won't have weird side effects?

Reply:
This is a safe default bc the method is only used to process incoming GTFS files. All the export methods allow for user selection if they want to export the pattern names and this just allows re-import